benchmark/babel-parser/real-case-ts/bench.mjs

```diff
@@ -28,10 +28,25 @@ function benchCases(name, implementation, options) {
     });
   }
 }
+copyFileSync(
+  require.resolve("@babel-baseline/parser"),
+  "./parser-baseline.cjs"
+);
+copyFileSync(
+  "../../../packages/babel-parser/lib/index.js",
+  "./parser-current.cjs"
+);
+
+const baseline = require("./parser-baseline.cjs");
+const current = require("./parser-current.cjs");
```

Comment:
I had to do this otherwise I would get unstable benchmarks for `current` and I don't understand why.
I suspect that might have something to do with it, but it's still weird. It shouldn't cause any effect.
![image](https://github.com/babel/babel/assets/30521560/b59cf5da-307a-4861-a743-9c6eb7ba81ce)